charts/gatekeeper-artifacts/crds/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
+!.gitkeep
```

Comment:
why are these here? Just a `*` would suffice

Reply:
I still don't get why you don't create a tmp chart with the example I gave you, so you:
- don't get into conflicts because of overwriting project files in docker mounts (when you thought of adding sudo)
- don't have to create incomplete charts that are not deployable on its own in the `charts/` folder.

I really start to get frustrated with you trying to avoid architectural decisions that have been made clear many times.

How much time have you wasted on this? I really want to hold you responsible from now on. I'd really like to see you align with solutions agreed upon.

Reply:
and why did you create a `.gitkeep` next to this `.gitignore`? Did you smoke too much? Please remove that

Reply:
Just a `*` will exclude the `.gitignore` file. Really now!

Reply:
> Just a `*` will exclude the `.gitignore` file. Really now!

Haha, no of course not. Why do you think I let you question your assumptions?

Reply:
> Did you smoke too much? Please remove that

While the assumption might be valid, can we please keep it civil? "While it might intimidate the reviewee into doing what you want, it’s not helpful in the long run - they will feel incapable, and there is not much info in there to help them improve."

Reply:
Agreed. Am resolving this conversation now as it starts to derail.